doc/deployment/openshift.md

```diff
@@ -144,9 +144,9 @@ In the deployment manifest, which we called `manifest.json`, above, find the sta
 		"ports": [
```

Comment:
I think this example may still be problematic because Kubernetes grabs the first entry in this list rather than the lowest port number. The change I used for getting the load test running was just swapping the first and second entry without changing the port numbers.